kmir/src/tests/integration/utils.py

```diff
@@ -100,3 +100,9 @@
 PROVE_FAIL_FILE = TEST_DATA_DIR / 'handwritten-prove-fail.tsv'
 PROVE_EXCLUDE = ['verify.k']
 PROVE_FAIL = {test.split('\t')[0] for test in PROVE_FAIL_FILE.read_text().splitlines() + PROVE_EXCLUDE}
+
+##--------------------
+# show-proof test data
+##--------------------
+SHOW_FILE = TEST_DATA_DIR / 'show-proof'
```

Comment:
It occurs to me that there is only one test for `show_proof`.
It is fine to have this single test.
Please create a seperate folder for `show_proof` tests related files.
SInce `verify.k` hardcoded the testcase (`num == 42`), it is fine to hardcode the expected result in the code.

Reply:
Well this is only just being integrated so there is one proof with the integration, but there will be more coming now this architecture is set up. Wouldn't it make sense to leave this so that it can be populated easily?

Reply:
Better to have a single test to start, and as we come up with more cases where proofs need to be demonstrated to show, then it's easy to add more. Always best to set-up a robust testing harness like this ahead of time so it's easier to add to it later if needed.